src/suparnaNJune20/Assignment_26/SumOFNoFromStr.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Assignment 26 : 9th July 2020
+WAP to sum all the numbers in given string.
+
+Input: P71Rr29T2E
+Ouput: Sum of all numbers in String is: 102
+
+Input: P6Rr29T2E3
+Ouput: Sum of all numbers in String is: 40
+ */
+package suparnaNJune20.Assignment_26;
+
+public class SumOFNoFromStr {
+	void findSum(String str) {
+			int sum = 0, tempNo = 0;
+		for (int outerIndex = 0; outerIndex < str.length(); outerIndex++) {
+			tempNo = 0;
+			if (Character.isDigit(str.charAt(outerIndex))) {
+				for (; outerIndex < str.length(); outerIndex++) {
+					if (Character.isDigit(str.charAt(outerIndex))) {
+						tempNo = tempNo * 10 + Character.getNumericValue(str.charAt(outerIndex));
+					} else {
+						sum = sum + tempNo;
+						break;
```

Comment:
Before break, you need to reset the tempNo to 0. and before printing the sum, need to check the tempNo value. 
try with this input: P171Rr29T2E >> output should be 202 but your logic returns 204